weekly_Q/hash_set.py

```diff
@@ -0,0 +1,49 @@
+class MyHashSet:
+
+    def __init__(self):
+        """
+        Initialize your data structure here.
+        """
+        self.keyspace = 3000
+        self.buckets = [[] for _ in range(self.keyspace)]
+
+
+    def add(self, key: int) -> None:
+        hash_key = key % self.keyspace
```

Comment:
Since you do the same calculation to get b here, and in remove() and in contains(), you should pull this out into its own method.